apps/codeforafrica/src/lib/index.js

```diff
@@ -171,7 +171,7 @@ async function getHomePageStaticProps() {
       title: "Code for Africa",
       sections: [
         {
-          ...getHero(),
+          ...getHero("index"),
```

Comment:
[`index`](https://github.com/CodeForAfrica/ui/blob/30e0fc31be6265fe87152bbbb3798188e5b00229/apps/codeforafrica/src/lib/api.netlify-cms/sections/getHero.js#L9) is the default though

Reply:
yeah. Just being more explicit. Maybe remove the default?

Reply:
You're the one who came up with the default if I remember correctly.

Reply:
yeah @kilemensi . Just rethinking the design a little bit.

Reply:
Sure Mr. API Design.